invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/RenameCommunitySlugButton.js

```diff
@@ -51,8 +51,8 @@ export class RenameCommunitySlugButton extends Component {
     try {
       await this.cancellableRename.promise;
 
-      window.location.href = `/communities/${newSlug}/settings`;
       this.handleClose();
+      window.location.href = `/communities/${newSlug}/settings`;
```

Comment:
Any special need for this? I would actually remove the `handleClose()`.
The thing is that the `refresh/redirect` takes a little bit of time to happen. You will see the modal closing, then wait like a sec, then redirect.
I would keep the modal open, in a loading state (buttons should not be clickable).

Reply:
Good point. Since this really only closes the modal, it should be okay to remove it